help/keyboard-shortcuts.md

```diff
@@ -156,6 +156,8 @@ in the Zulip app to add more to your repertoire as needed.
   src="/static/generated/emoji/images/emoji/unicode/1f44d.png"
   title="thumbs up"/>**: <kbd>+</kbd>
 
+* **Toggle first emoji reaction on selected message**: <kbd>=</kbd>
+
```

Comment:
I think we can drop "on selected message" here, as it's indicated in the section heading, and that will be more consistent with other instructions.

Reply:
This fix got put in the wrong commit; I fixed that while reviewing.